mirdata/datasets/mtg_jamendo_autotagging_moodtheme.py

```diff
@@ -0,0 +1,340 @@
+    """Load a MTG_jamendo_autotagging_moodtheme audio file.
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to an audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=None, mono=False)
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The MTG_jamendo_autotagging_moodtheme dataset
+    """
+
+    def __init__(self, data_home=None, version="default"):
+        super().__init__(
+            data_home,
+            version,
+            name="mtg_jamendo_autotagging_moodtheme",
+            track_class=Track,
+            bibtex=BIBTEX,
+            download_info=DOWNLOAD_INFO,
+            indexes=INDEXES,
+            remotes=REMOTES,
+            license_info="Creative Commons Attribution NonCommercial Share Alike 4.0 International.",
+        )
+
+    @core.cached_property
+    def _metadata(self):
+        meta_path = os.path.join(self.data_home, "data/autotagging_moodtheme.tsv")
+        if not os.path.exists(meta_path):
+            raise FileNotFoundError("Metadata not found. Did you run .download()?")
+
+        with open(meta_path, "r") as fhandle:
+            d = list(
+                csv.DictReader(
+                    fhandle,
+                    delimiter="\t",
+                    fieldnames=[
+                        "TRACK_ID",
+                        "ARTIST_ID",
+                        "ALBUM_ID",
+                        "PATH",
+                        "DURATION",
+                        "TAGS",
+                    ],
+                )
+            )
+            meta = {
+                m["TRACK_ID"]: {
+                    "ARTIST_ID": m["ARTIST_ID"],
+                    "ALBUM_ID": m["ALBUM_ID"],
+                    "PATH": m["PATH"],
+                    "DURATION": m["DURATION"],
+                    "TAGS": m["TAGS"],
+                }
+                for m in d[1:]
+            }
+
+        splits = []
+        for ii in range(5):
+            split = {}
+            path_train = os.path.join(
+                self.data_home,
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-train.tsv",
+            )
+            with open(path_train, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["train"] = [m["TRACK_ID"] for m in d[1:]]
```

Comment:
```suggestion
                reader = csv.reader(fhandle, delimiter="\t")
                track_uris = [line[0] for line in reader]
                split["train"] = track_uris[1:]
```

and the same for the lines below